tests/integration/test_parallel_replicas_custom_key/test.py

```diff
@@ -161,6 +161,9 @@ def test_parallel_replicas_custom_key_replicatedmergetree(
 
     insert_data("test_table_for_rmt", row_num, all_nodes=False)
 
+    for node in nodes:
```

Comment:
Maybe we could put this into `insert_data`. Although I don't see the point of having this `insert_data` function, when it's called only twice and has a conditional based on the input :sweat: